bio2zarr/tskit.py

```diff
@@ -111,25 +131,11 @@ def iter_field(self, field_name, shape, start, stop):
         else:
             raise ValueError(f"Unknown field {field_name}")
 
-    def iter_alleles(self, start, stop, num_alleles):
-        for variant in self.ts.variants(
-            samples=self.sample_ids,
-            isolated_as_missing=self.isolated_as_missing,
-            left=self.positions[start],
-            right=self.positions[stop] if stop < self.num_records else None,
-        ):
-            alleles = np.full(num_alleles, constants.STR_FILL, dtype="O")
-            for i, allele in enumerate(variant.alleles):
-                assert i < num_alleles
-                alleles[i] = allele
-            yield alleles
-
     def iter_alleles_and_genotypes(self, start, stop, shape, num_alleles):
-        # In tskit, all genotypes are considered phased
+        # All genotypes in tskit are considered phased
         phased = np.ones(shape[:-1], dtype=bool)
 
         for variant in self.ts.variants(
-            samples=self.sample_ids,
```

Comment:
Have to pass samples somehow, otherwise we can't get data for non-sample nodes out.